.php_cs

```diff
@@ -1,18 +1,23 @@
 <?php
```

Comment:
Is this file generated by swagger-generator? Can you adjust the rules? Because code sniffer gives many errors and warnings. Not that this is important.